timemachine/fe/free_energy.py

```diff
@@ -376,7 +411,7 @@ def sample(
     rng = np.random.default_rng(md_params.seed)
 
     if md_params.local_steps > 0:
-        ctxt.setup_local_md(initial_state.integrator.temperature, md_params.freeze_reference)
+        ctxt.setup_local_md(temperature, md_params.freeze_reference)
```

Comment:
Think this will raise an exception on the second call. Probably worth just making this idempotent

Reply:
I will put together a PR for this, to avoid adding unrelated cruft to this PR.